rust-toolchain

```diff
@@ -5,6 +5,6 @@
 # to the user in the error, instead of "error: invalid channel name '[toolchain]'".
 
 [toolchain]
-channel = "1.65"
+channel = "1.70"
```

Comment:
Oof this is tough - we did this so that people would be using the proper MSRV for wgpu (and not get errant clippy warnings), but we still can't actually bump our MSRV appreciably.

Reply:
Yes; it annoyed me s well, however some of the deno deps use features from 1.70 :/

Reply:
does it maybe work to have toolchains for the 2 deno folders? not sure if thats possible

Reply:
I don't think so. Let's bump this for now, but keep MSRV testing in CI. There's probably a better solution out there.